### ISSUE TITLE
`pa11y-ci` fixes for concurrency and runner problems

### ISSUE THREAD

**Issue by DaveSkender** [18F/federalist-uswds-jekyll#225]
`pa11y-ci` accessibility tests fail intermittently.  See https://github.com/pa11y/pa11y-ci/issues/168 for more information.  If this is a persistent problem, we'll need to fix this here too so it doesn't propagate to new Federalist sites.

https://github.com/18F/federalist-uswds-jekyll/blob/f543121a3578b84f72d9235eb43744fb0540d033/.pa11yci#L1-L7

My workaround is to change the configuration in `.pa11yci` to:
```json
{
  "defaults": {
    "concurrency": 1,
    "standard": "WCAG2AA",
    "runners": ["htmlcs"]
  }
}
```

**Comment by DaveSkender** [comment]
This issue _might_ be fixed in v3.0.1, so an upgrade potentially supersedes this workaround.
